src/rpc/handlers/LedgerEntry.hpp

```diff
@@ -101,6 +102,7 @@ class LedgerEntryHandler {
         std::optional<uint32_t> chainClaimId;
         std::optional<uint32_t> createAccountClaimId;
         std::optional<ripple::uint256> oracleNode;
+        std::optional<bool> include_deleted;
```

Comment:
1 Maybe we can use bool directly? False by default? 
 2 Name convention: includeDeleted